src/main/webapp/app/exam/participate/exam-navigation-bar/exam-navigation-bar.component.ts

```diff
@@ -54,6 +65,31 @@ export class ExamNavigationBarComponent implements OnInit {
                 this.itemsVisiblePerSide = 0;
             }
         });
+
+        const isInitialSession = this.examSessions && this.examSessions.length > 0 && this.examSessions[0].initialSession;
+        if (isInitialSession || isInitialSession == undefined) {
+            return;
```

Comment:
I think this is a bit dangerous, a later developer might overlook this, add some more initalization below and miss the fact the code might not get executed. I would prefer wrapping the call below into the if instead of returning early.

Reply:
Hey @MDK5 thank you for the review 😃 
As I already mentioned in the testing session, I don't find it a good coding style to have a lot of levels of nested ifs.
I think early returns are great and should be practiced as it makes it easier to follow the "success path" in code.
Regarding developers overseeing it, I would say that for this we have tests.